src/PHPRenderer.php

```diff
@@ -67,6 +67,8 @@ public function render(ResponseInterface $response, $template, array $data = [])
         $render($this->templatePath . $template, $data);
         $output = ob_get_clean(); 
 
-        return $response->getBody()->write($output);
+        $response->getBody()->write($output);
```

Comment:
You can change that line too... 
$response = $response->getBody()->write($output);